install.sh

```diff
@@ -48,7 +48,7 @@ then
   HOMEBREW_CACHE="${HOME}/Library/Caches/Homebrew"
   HOMEBREW_CORE_DEFAULT_GIT_REMOTE="https://github.com/Homebrew/homebrew-core"
 
-  STAT="stat -f"
+  STAT_FLAG="-f"
   PERMISSION_FORMAT="%A"
   CHOWN="/usr/sbin/chown"
   CHGRP="/usr/bin/chgrp"
```

Comment:
```suggestion
  STAT=("stat" "-f")
  PERMISSION_FORMAT="%A"
  CHOWN=("/usr/sbin/chown")
  CHGRP=("/usr/bin/chgrp")
```

Similarly, change `TOUCH` to an array below.

Then replace all `${STAT}` to `${STAT[@]}` in this file. Similar changes for `CHOWN`, `CHGRP`, and `TOUCH`.